pkg/network/infraconfigurators/infraconfigurators_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package infraconfigurators_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestInfraconfigurators(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Infraconfigurators Suite")
+}
```

Comment:
Can you change the content of this file to 

```go
import (
	"testing"

	"kubevirt.io/client-go/testutils"
)

func TestHeartbeat(t *testing.T) {
	testutils.KubeVirtTestSuiteSetup(t)
}
```

Then the unit-tests shhould succeed.